cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,16 @@
+import {Upload} from "../page/index";
+
+describe("uploading a file test", () => {
+  let upload: Upload;
+
+  before(() => {
+    upload = new Upload();
+  });
+
+  it("testing the title after upload file", () => {
+    const data = ["example.json"];
+    upload.visitWebSite();
+    upload.uploadFile("../e2e/fixtures/example.json");
```

Comment:
if you move the fixtures to the original path: `./cypress/fixtures`. Then the attach file method within the upload page object will look the "example.json" file there, and you will not need to specify the path to the fixtures folder